lms/djangoapps/grades/tests/test_models.py

```diff
@@ -238,7 +238,6 @@ def test_optional_fields(self):
     @ddt.data(
         ("user_id", ValidationError),
         ("usage_key", KeyError),
-        ("subtree_edited_timestamp", ValidationError),
```

Comment:
Update `test_optional_fields` to ddt with `subtree_edited_timestamp` and `course_edited_timestamp`?